src/api/cristinApi.ts

```diff
@@ -121,24 +121,28 @@ export const fetchPerson = async (personId: string) => {
 
 export interface PersonSearchParams {
   name?: string | null;
+  orderBy?: string | null;
```

Comment:
kan godt angi i typen her hvilke felter vi støtter sortering på
```suggestion
  orderBy?: 'name' | null;
```

Reply:
eller, kanskje dette gir litt kluss over, ved at vi må caste typer osv, så kanskje like greit å bare droppe det også!

Reply:
Gjorde faktisk slik som du foreslår først, men så ble det akkurat som du sier at det ble mye kluss andre plasser (bl.a. urlParams.get(...) måtte castes osv). Så jeg dropper det :)